app/controllers/application_controller.rb

```diff
@@ -137,4 +138,20 @@ def sample_requests_for_scout
     #   ScoutApm::Transaction.ignore!
     # end
   end
+
+  def set_ngrok_urls
+    if Ngrok::Tunnel.running?
+      # Getting current url
+      url = Ngrok::Tunnel.ngrok_url_https
+
+      # Variable hash
+      default_url_options = {host: url}
+
+      # Overwriting current variables
+      Rails.application.config.action_controller.asset_host = url
+      Rails.application.config.action_mailer.asset_host = url
+      Rails.application.routes.default_url_options = default_url_options
+      Rails.application.config.action_mailer.default_url_options = default_url_options
+    end
+  end
```

Comment:
I wonder if this would be better in an initializer. Would possibly require a restart if/when devs enable ngrok, but I think that would be ok.

Reply:
I'll hold off on this till you can work your magic